### DISCUSSION TITLE
Is there an attribute that tracks how much data has been trained on?

### DISCUSSION THREAD

**Discussion by jpfeil** [online-ml/river#1467] [Q&A] [answered]
Sorry if I'm missing it. I basically want to know if the model or metrics keep track of the number of data that has been trained on so far. If not, I'll keep track of it in my script. But just thought this might already be an attribute.

**Comment by MaxHalford** [comment, marked as answer]
Hey! No there isn't. Sometimes metrics that it, but that's all. And as you point out, it's not difficult for you to do it yourself. Always good to check if there's idiomatic way ;)
